autometa/common/external/work_queue.py

```diff
@@ -179,14 +179,13 @@ def main(args):
     parser.add_argument('password', help='</path/to/work-queue/master/password.txt>')
     parser.add_argument('project-name', help='<unique project identifier>')
     # TODO: Allow argparse to handle range of ints for port
-    parser.add_argument('--port', help="""
-    The port number to listen on. If zero, then a random port is chosen. A range
-     of possible ports (low, hight) can be also specified instead of a single integer.
-    """,
-        default=wq.WORK_QUEUE_DEFAULT_PORT, type=int)
+    #parser.add_argument('--port', help="""
+    #The port number to listen on. If zero, then a random port is chosen. A range
+    # of possible ports (low, hight) can be also specified instead of a single integer.
+    #""",
+    #    default=wq.WORK_QUEUE_DEFAULT_PORT, type=int)
```

Comment:
I am getting an error in the `parse_argparse.py` if I un-comment it:
`File "/home/siddharth/Autometa/docs/temp_write.py", line 19, in <module>
    default=wq.WORK_QUEUE_DEFAULT_PORT, type=int)
NameError: name 'wq' is not defined`

Reply:
I did what you suggested earlier:

`default=wq.WORK_QUEUE_DEFAULT_PORT, type=int)
# becomes:
default="wq.WORK_QUEUE_DEFAULT_PORT", type=int)`

Fortunately, `default=int` is not causing any problems

Reply:
Interesting. Wrapping the default in a string is nice, because we still see how the default is being populated. Does argparse give an error to this? Because this is displaying conflicting arguments.

Namely, the type is supposed to be an integer (`int`) and you are providing a string default type (`str`).  🤔